rid/v1/observation.yaml

```diff
@@ -122,12 +126,33 @@ components:
         timestamp:
           description: Time at which the message containing the current state information was generated by the display provider.
           type: string
+        timestamp_accuracy:
+          format: float
+          description: 'Declaration of timestamp accuracy, which is the largest difference
+            between Timestamp and true time of applicability for any of the following
+            fields: Latitude, Longitude, Geodetic Altitude, Pressure Altitude of Position,
+            Height. to determine time of applicability of the location data provided.  Expressed
+            in seconds, precise to 1/10ths of seconds. The accuracy reflects the 95%
+            uncertainty bound value for the timestamp.'
+          minimum: 0
+          exclusiveMinimum: false
+          type: number
+          default: 0
         operational_status:
           $ref: 'commons.yaml#/components/schemas/RIDOperationalStatus'
         track:
           $ref: 'commons.yaml#/components/schemas/RIDTrack'
         speed:
           $ref: 'commons.yaml#/components/schemas/RIDSpeed'
+        speed_accuracy:
+          anyOf:
+            - $ref: 'commons.yaml#/components/schemas/SpeedAccuracy'
+          description: Accuracy of horizontal ground speed.
+        vertical_speed:
+          format: float
+          description: Speed up (vertically) WGS84-HAE, m/s.
+          type: number
+          example: 0.2
         # TODO for next version: Move most_recent_position here.
```

Comment:
@barroco I ran into this TODO: I don't think it is a priority for the current NetRID effort, but if it has any importance, doing the move could be done along this change in a separate PR.